src/mapping/cellranger_multi/test.py

```diff
@@ -187,5 +187,59 @@ def test_cellranger_multi_crispt_data(run_component):
     # check for crispr data
     assert Path("output5/per_sample_outs/run/count/crispr_analysis/").is_dir()
 
+def test_cellranger_multi_helper_input(run_component):
```

Comment:
What happens if an empty directory is passed to `--input` with no 'helper' inputs?

Reply:
Perhaps it is a good idea to check if an directory is passed to check if that directory contains any files matching to the regex?

Reply:
This is checked under the function where paths are resolved to files: https://github.com/openpipelines-bio/openpipeline/blob/3b3fde4909925bc38e6b1e2e7e9b5f13363f1db7/src/mapping/cellranger_multi/script.py#L194

I just added this additional assert to make sure that input is provided, because in the viash config none of the input fields are required anymore...

Reply:
Nice!